operator/cilium_node.go

```diff
@@ -30,7 +30,15 @@ import (
 	"k8s.io/client-go/tools/cache"
 )
 
-var k8sCiliumNodesCacheSynced = make(chan struct{})
+var (
+	// CiliumNodeStore contains all CiliumNodes present in k8s.
+	// Warning: The CiliumNodes stored in the cache are not intended to be
+	// used for Update operations in k8s as some of its fields were are not
+	// populated.
+	CiliumNodeStore cache.Store
```

Comment:
Does this need to be exported? It doesn't look like it's used outside of the `operator` package.

```suggestion
	
	ciliumNodeStore cache.Store
```

Reply:
Done.